### DISCUSSION TITLE
How to do math expression in Nikola?

### DISCUSSION THREAD

**Discussion by quanta-science** [getnikola/nikola#3576] [Q&A] [answered]
Hi guys, I used HTML input formats (not reST) to generate a test html-page with some math expressions but I failed with all my tries. At the beginning I used \( delimiters for inline and \[ delimiters for display with no changes in conf.py but I saw only such things like this "\int \frac{dx}{1+ax}=\frac{1}{a}\ln(1+ax)+C" on the page after "nikola build" command. Then I used all variations of delimiters and changes in conf.py. Nothing works. Who knows what to do? your help is very appreciated.

p.s. I cannot use reST format because of my tables with sticky properties in their heads (and others which are possible to make only in CSS).

**Comment by Kwpolska** [comment, marked as answer]
Does your post/page have `.. has_math: yes` in the meta section? Specifying this is required for math to work. I was able to get math working in a HTML post with `\(foo\)` and `..has_math: yes`.

**Reply by quanta-science** [reply]
Thank you very much, I need to read metadata section more deep after your answer.
